# The following output should be displayed, when running this check (tested in MacOS)
'''
Traceback (most recent call last):
  File "name-main_check.py", line 8, in <module>
    from Port_Scan import name_scan
ImportError: cannot import name 'name_scan'
'''
from Port_Scan import name_scan